projects/kit/components/input-time/native-time/native-time.component.ts

```diff
@@ -0,0 +1,65 @@
+import {ChangeDetectionStrategy, Component, Inject} from '@angular/core';
+import {TuiIdService} from '@taiga-ui/cdk';
+import {TUI_TEXTFIELD_HOST} from '@taiga-ui/core';
+
+import type {TuiInputTimeDirective} from '../input-time.directive';
+
+@Component({
+    selector: 'input[tuiTime]',
+    template: `
+        <datalist
+            *ngIf="items.length"
+            [id]="autoIdString"
+        >
+            <option
+                *ngFor="let item of items"
+                value="{{ item }}"
+            ></option>
+        </datalist>
+    `,
+    host: {
+        type: 'time',
+        '[attr.list]': 'autoIdString',
+        '[tabIndex]': '-1',
+        '[value]': 'value',
+        '[step]': 'step',
+        '(change.stop.silent)': 'onChange($event.target.value)',
```

Comment:
Should it be `silent`? Change detection seems to not trigger for items picked from the list.